test/cases/5.6_string_manipulation.yaml

```diff
@@ -259,6 +259,33 @@ tests:
     expression: Functions.str.empty.toChars()
     result: []
 
+  - desc: '** split string on delimiter'
```

Comment:
I think the tests here should include the [tests](https://github.com/HL7/FHIRPath/pull/10/files#diff-88808fa32d38b9c51132f29918649ca83d25b1a517f704fd6aa43d7af8b191eaR643) in the FHIRPath PR for these functions.